lib/template.hb.swift

```diff
@@ -6,35 +11,20 @@ extension Stylist {
 {{#if styles}}
 {{#each styles}}
         stylist.addStyle(identifier: "{{name}}") { 
-            AnyView($0{{#each style}}
-                      .{{name}}({{{value}}}){{/each}}
-                      {{#if text}}
+            AnyView($0{{#if text}}
                       .styleText { text in
                           text{{#each text}}
                               .{{name}}({{{value}}}){{/each}}
                       }{{/if}}
+                      {{#each style}}
+                        .{{name}}({{{value}}})
```

Comment:
Is there any particular reason to swap these around?

Reply:
Doesn't compile otherwise -- `styleText` isn't available on the result of a regular `ViewModifier` only on the type that's passed in (before applying other modifiers)